packages/hiro-graph-client/index.d.ts

```diff
@@ -448,6 +448,97 @@ export interface VariablesServlet {
   define<T = any>(options: DefineVariableOptions): Promise<T>;
 }
 
+export type ActionLogType =
+  | 'SearchKI'
+  | 'SearchTask'
+  | 'SearchTeachingSession'
+  | 'DeployKI'
+  | 'UndeployKI'
+  | 'WriteKI'
+  | 'ForkKI'
+  | 'OpenTeachingSession'
+  | 'OwnTeachingSession'
+  | 'StartTeachingSession'
+  | 'ContinueTeachingSession'
+  | 'StartConversionSession'
+  | 'ContinueConversionSession'
+  | 'AssignKI';
+
+export interface BaseEvent<T extends ActionLogMeta> {
+  action: ActionLogType;
+  instanceId: string;
+  meta: T;
+}
+
+export interface SearchEventMeta {
+  searchString: string;
+}
+
+export interface KIEventMeta {
+  kiId: string;
+  kiName: string;
+}
+
+export interface TeachingSessionEventMeta {
+  sessionId: string;
+  sessionName: string;
+  taskId: string;
+  desktopRole: string;
+}
+
+export interface DeployKIEventMeta {
+  knowledgePoolId: string;
+  knowledgePoolName: string;
```

Comment:
`kiId` and `kiName` are needed here as well

Reply:
Thanks, done!